allennlp/tests/data/tokenizers/pretrained_transformer_tokenizer_test.py

```diff
@@ -169,99 +160,59 @@ def test_token_idx_roberta(self):
             "<s>",
             "A",
             ",",
-            "ĠnaÃ¯ve",  # RoBERTa has a funny way of encoding combining characters.
+            "naïve",
```

Comment:
I wonder if this breaks RoBERTa usage, because the tokenizer is what changed but the model is the same.

Reply:
RoBERTa usage is broken because of the leading spaces. The special characters are now handled the right way (except where they break the offsets).